src/plotting/task_plot_spring_incidences.py

```diff
@@ -0,0 +1,27 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import pytask
+import seaborn as sns
+
+from src.config import BLD
+from src.config import POPULATION_GERMANY
+from src.plotting.plotting import style_plot
+
+
+@pytask.mark.depends_on(BLD / "data" / "processed_time_series" / "rki.pkl")
+@pytask.mark.produces(BLD / "data" / "processed_time_series" / "spring_incidences.png")
+def task_plot_spring_incidences(depends_on, produces):
+    rki = pd.read_pickle(depends_on)
+    rki_n_cases = rki.groupby("date")["newly_infected"].sum()
+    rki_incidence = rki_n_cases * 100_000 / POPULATION_GERMANY
+    smoothed_incidence = rki_incidence.rolling(7, center=True).sum()
```

Comment:
otherwise the line anticipates changes. Also, can't we use the calc_moments function for this to have the logic in one central space?

```suggestion
    smoothed_incidence = rki_incidence.rolling(7, center=False).sum()
```